client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -59,8 +61,26 @@ const DashboardHeading = styled.h1`
 	flex: 1;
 `;
 
-export function SitesDashboard( { queryParams: { search, status = 'all' } }: SitesDashboardProps ) {
+const HiddenSitesMessage = styled.span`
+	color: var( --color-text-subtle );
+	display: block;
+	font-size: 14px;
+	padding: 16px 16px 24px;
```

Comment:
Personally I prefer 
```suggestion
	padding: 16px 0 24px 0;
```

Which aligns the left of the text with the left of the rows.

![CleanShot 2022-08-15 at 14 56 18@2x](https://user-images.githubusercontent.com/1500769/184569724-0e09162d-e74c-4a86-9ea5-e921fe176322.png)